stdlib/Logging/src/ConsoleLogger.jl

```diff
@@ -127,7 +128,14 @@ function handle_message(logger::ConsoleLogger, level, message, _module, group, i
 
     # Format lines as text with appropriate indentation and with a box
     # decoration on the left.
-    color,prefix,suffix = logger.meta_formatter(level, _module, group, id, filepath, line)
+    format = logger.meta_formatter(level, _module, group, id, filepath, line)
+    if length(format) == 4
+        prefix_color, prefix, suffix_color, suffix = format
+    else
+        # The three-value version of meta_formatter should be depreciated at some point
```

Comment:
AFAIK this is an unexported API function and thus this change can go into Julia 1.1 or the next minor bump of the Logging package.

This should also have an explicit deprecation message for the 3 value version, so developers can plan for this change appropriately

Reply:
It depends how you define public API. The docstring for `ConsoleLogger` mentions the `meta_formatter` option, which I would argue makes it part of the public API (even if the default meta formatter is unexported).